src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -284,16 +284,15 @@ public static boolean isFunctionTranslatedToScript(String function) {
                 functionStr = log(SQLUtils.toSQLExpr("10"), (SQLExpr) paramers.get(0).value, name);
                 break;
             case "log":
-                List<SQLExpr> logs = Lists.newArrayList();
-                for (int i = 0; i < paramers.size(); i++) {
-                    logs.add((SQLExpr) paramers.get(0).value);
-                }
-                if (logs.size() > 1) {
-                    functionStr = log(logs.get(0), logs.get(1), name);
+                if (paramers.size() > 1) {
+                    functionStr = log((SQLExpr) paramers.get(0).value, (SQLExpr) paramers.get(1).value, name);
```

Comment:
should this be parameters instead of paramers?

Reply:
We changed the code on purpose cuz any time LOG is called the previous code would create a new list to store the paramers, which is not necessary. Thanks!